src/editor/map/map_context.hpp

```diff
@@ -25,8 +25,9 @@
 #include "team.hpp"
 #include "tod_manager.hpp"
 #include "units/map.hpp"
-#include <optional>
+#include "string_enums/side_controller.hpp"
```

Comment:
Please put this one in its alphabetical place too.

Reply:
Done.